python_modules/dagstermill/dagstermill/examples/repository.py

```diff
@@ -118,6 +120,46 @@ def define_test_notebook_dag_pipeline():
     )
 
 
+DataFrame = as_dagster_type(pd.DataFrame)
```

Comment:
import the DataFrame type from dagster_pandas rather than rolling your own. you've lost input schema etc